js/scheduler.js

```diff
@@ -365,10 +365,7 @@
 
 			var data = {
 				startDateTime: startDateTime,
-				timeZone: {
-					name: timeZone.name,
-					offset: timeZone.offset
-				},
+				timeZone: timezone,
```

Comment:
Looking through this file I don't see where `timezone` (all lower case is being set). Please confirm.